src/async-iterator.ts

```diff
@@ -2,18 +2,21 @@ import { PubSubEngine } from 'graphql-subscriptions'
 import { $$asyncIterator } from 'iterall'
 
 export class PubSubAsyncIterator<T> implements AsyncIterator<T> {
+  private readonly createdAt = new Date().getTime()
```

Comment:
```suggestion
  private readonly createdAt = Date.now()
```

I think you could just use this

Reply:
I can try. But still there is problem, when your backend environment has wrong time. The only one solution which can fix this is to get date from firebase. But there is no simple solution to get it.

How to do it:
1. create and save object {timestamp:ServerValue.TIMESTAMP}
2. read the timestamp value
3. save it to createAt

But this is overkill. What you think?

Reply:
Or to use Date.now() instead of ServerValue.TIMESTAMP everywhere

Reply:
@vrehor yeah I think let's just swap out `ServerValue.TIMESTAMP` for `Date.now()` on L51 in `pubsub.ts` like you suggested - That should probably give as good of a time value anyways

Alternatively you can allow a bit of grace period, so `createdAt = Date.now() - 1000` or something like that in L5 of async-iterator.ts?

I think either of those solutions would be better than trying to read back the value of ServerValue.TIMESTAMP, feel free to take whichever you prefer :)

Reply:
@swantzter I picked the first one. Now it is synchronized with getDeletionRoutineFunction where Date.now function is also used :-)